package-lock.json

```diff
@@ -7,26 +7,27 @@
     "": {
```

Comment:
I worry that all the changes in this file could cause issues, but I will still approve and we will find out.

Reply:
From my research, package-lock.json would "lock" the latest version. But in this place, we use legacy versions. Hence, if we want to upgrade them, I have to remove this one so that npm could install the right version. Otherwise, it might always install the latest one, which is not we want for this version.

Reply:
So what you're saying is that we should remove this file entirely?